report.go

```diff
@@ -112,8 +113,17 @@ func writeStack(w io.Writer) {
 	thisPackage := reflect.TypeOf(C{}).PkgPath() + "."
 	for {
 		frame, more := frames.Next()
-		if strings.HasPrefix(frame.Function, "testing.") || strings.HasPrefix(frame.Function, thisPackage) {
-			// Do not include stdlib test runner and quicktest checker calls.
+		if strings.HasPrefix(frame.Function, "testing.") {
+			// Stop before getting back to stdlib test runner calls.
+			break
+		}
+		if strings.HasPrefix(frame.Function, thisPackage) {
+			if r := rune(frame.Function[len(thisPackage)]); unicode.IsUpper(r) {
+				// Continue without printing frames for quicktest exported API.
```

Comment:
or you can just use https://golang.org/pkg/go/ast/#IsExported, which is also self-explanatory

Reply:
maybe:

     if ident := strings.TrimPrefix(frame.Function, thisPackage); ident != frame.Function {
         if ast.IsExported(ident) {
             continue
         }

?

Reply:
Good idea!